plentyofpets/lib/screens/add_edit_pet_screen.dart

```diff
@@ -0,0 +1,188 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+class AddPetScreen extends StatefulWidget {
+  const AddPetScreen({Key? key}) : super(key: key);
+  @override
+  _AddPetScreenState createState() => _AddPetScreenState();
+}
+
+
+class _AddPetScreenState extends State<AddPetScreen> {
+  final _petFormKey = GlobalKey<FormBuilderState>();
+  final List<String> availableOptions=['Not Available','Available','Pending','Adopted'];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text('Pet Information'),
+        ),
+        body: SingleChildScrollView(
+          child: Padding(
+            padding: const EdgeInsets.all(25.0), 
+            child: FormBuilder(
+              key: _petFormKey, 
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch, 
+                children: <Widget>[
+                  
+                  const Text(
+                    'Pet Profile Form',
+                    textAlign: TextAlign.center,
+                    style: TextStyle(
+                      color: Colors.black,
+                      fontSize: 40,
+                      fontWeight: FontWeight.bold
+                    ),
+                  ),
+
+                  const Text(
+                    'Please complete the information to include in your pet\'s adoption profile',
+                    textAlign: TextAlign.center,
+                    style: TextStyle(
+                      color: Colors.blue,
+                      fontSize: 20,
+                      fontWeight: FontWeight.bold
+                    ),
+                  ),
+                  
+                  const Padding(padding: EdgeInsets.fromLTRB(0,5,0,35),),
+
+                  FormBuilderChoiceChip(
+                    name: 'petType',
+                    decoration: const InputDecoration(
+                      labelText: 'Pet Type:',
+                      labelStyle: TextStyle(
```

Comment:
I think the form is lookin great so far. My only major comment is that I think we should move this TextStyle into the top level theme.dart file that I added. That way we don't have to repeat it everywhere in this file. There is also a TextStyle with just fontSize: 15 here that could be moved into the Themes as well.

Reply:
I'm not as familiar with themes as I'd like to be at this point.  I'll definitely look into that when we finalize the look and style of all the screens.  As this is just a rough version of my screen, I'll keep the code as is for now and we can come back to styling when we revisit that as a group.